css/style.css

```diff
@@ -0,0 +1,79 @@
+body{
+  background-color: #f3f4f5;
+  padding: 15px;
+  font-family: Helvetica;
+  font-size: 14px;
+  background-image: url(http://www.gannett-cdn.com/-mm-/d8b68461f8c54f619579b4b1091145ba082f9ab2/c=0-1287-4000-3547&r=x1683&c=3200x1680/local/-/media/2016/05/08/USATODAY/USATODAY/635983284547965765-Radio-AMoon-Cover-240416.jpg);
+  background-repeat: no-repeat;
+  background-position: absolute;
```

Comment:
@jcooper8190 There is not `background-position: absolute`, You're probally confused it with `position: absolute` which is used for positioning elements.